tests/components/media_player/test_cast.py

```diff
@@ -392,8 +394,11 @@ def test_stop_discovery_called_on_stop(hass):
             return_value=mock_coro()) as mock_setup:
         await cast.async_setup_entry(hass, MockConfigEntry(), None)
 
+    print(mock_setup.mock_calls[0])
```

Comment:
Stale debug print.

Reply:
That line isn't mine actually... Removed it